packages/hydrogen/src/customer/auth.helpers.ts

```diff
@@ -0,0 +1,253 @@
+export const userAgent =
+  'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.93 Safari/537.36';
```

Comment:
I wonder if we just put the user agent as "Shopify Hydrogen" or something.

Reply:
Yeah, this could potentially help troubleshoot these requests in the future